name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/dialogs/ListSelectionDialog.java

```diff
@@ -172,11 +172,14 @@ protected Control createDialogArea(Composite parent)
 
         elementFilter = new ElementFilter();
 
-        int style = SWT.BORDER | SWT.FULL_SELECTION;
+        int style = SWT.BORDER | SWT.FULL_SELECTION | SWT.VIRTUAL;
         if (isMultiSelection)
```

Comment:
At least for other VIRTUAL tables, I also needed this config:

```java
tableViewer.setUseHashlookup(true);
```

Otherwise there were strange error messages on some platforms. So I propose to add it here as well.